server/src/main/java/org/apache/druid/rpc/StandardRetryPolicy.java

```diff
@@ -141,6 +141,7 @@ public boolean retryHttpResponse(final HttpResponse response)
     return code == HttpResponseStatus.BAD_GATEWAY.getCode()
            || code == HttpResponseStatus.SERVICE_UNAVAILABLE.getCode()
            || code == HttpResponseStatus.GATEWAY_TIMEOUT.getCode()
+           || code == HttpResponseStatus.CONFLICT.getCode()
```

Comment:
I am not sure if retrying whenever a conflict response is received is a good idea.
We don't know if the conflict will go away on retrying.